src/components/GlobalPreferences/CustomLabels/LocalIdentities.js

```diff
@@ -53,7 +53,7 @@ function LocalIdentities({
   const theme = useTheme()
 
   return (
-    <Box>
+    <Box padding={3 * GU}>
```

Comment:
Let's do this for now... but we should update Box in aragonUI to do this automatically cc @bpierre.

Reply:
Let's make it very explicit in the docs for other devs that `Box` will have different spacing all around:
- For smaller views (mostly mobile `8px`)
- For larger views when in `Split` (`24px`)
- For larger views when not in `Split` (`40px`)